community/cloud-foundation/templates/cloudbuild/trigger.py.schema

```diff
@@ -0,0 +1,429 @@
+# Copyright 2018 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+info:
+  title: Cloud Build Trigger
+  author: Sourced Group Inc.
+  description: |
+    Supports creation of an automated Cloud Build trigger.
+    For more information on this resource, see
+    https://cloud.google.com/cloud-build/docs/running-builds/automate-builds.
+
+required:
+  - triggerTemplate
+
+properties:
+  description:
+    type: string
+    description: The human-readable trigger description.
+  triggerTemplate:
+    type: object
+    description: |
+      The template describing the types of source changes that trigger a build.
+      Branch and tag names in the trigger templates are interpreted as regular
+      expressions. Any branch or tag change that matches that regular
+      expression triggers a build.
+    properties:
+      projectId:
+        type: string
+        description: |
+          The ID of the project that owns the Cloud Source repository.
+          If omitted, the IF of the project requesting the build is assumed.
```

Comment:
Did you mean ID?

Reply:
`projectId` is the correct field (this is optional)